README.md

```diff
@@ -25,73 +25,85 @@ When we say expressive, we mean it: math, concatenation, regex, conditionals, fu
 
 ## Features
 
-* Modifier styles based on component props
-* Media queries
+* Conceptually simple extension of normal inline styles
 * Browser state styles to support `:hover`, `:focus`, and `:active`
-* Dynamic computed styles
+* Media queries
 
 ## Docs
 
-- [Guides](docs/guides)
-  - [Overview](docs/guides/overview.md)
-  - [Media Queries](docs/guides/media-queries.md)
-  - [Computed Styles](docs/guides/computed-styles.md)
+- [Overview](docs/guides)
 - [API Docs](docs/api)
 
 ## Usage
 
-Start by writing a style object with a combination of default styles, browser states, media queries, and modifiers. Pass the object to `this.buildStyles()` and Radium will determine the correct group of style rules to apply to your component.
+Start by adding `Radium.wrap()` around the config you pass to `React.createClass`. Then, write a style object as you normally would with inline styles, and add in styles for interactive states and media queries. Pass the style object to your component via `style={...}` and let Radium do the rest!
 
 ```xml
-<Button kind='primary'>Radium Button</Button>
+<Button kind="primary">Radium Button</Button>
```

Comment:
Oh jeez. Interesting re: Facebook convention. I'd prefer to stick with all single quotes everywhere, rather than using a different type depending on the context.

That said, this is a nit that can be addressed later :+1: